dela.sh

```diff
@@ -0,0 +1,38 @@
+source .env;
```

Comment:
I'd suggest using a different filename to avoid confusion, especially as it's docker-specific.